lib/generators/linked_data_fragments/templates/config/ldf.yml

```diff
@@ -0,0 +1,21 @@
+development:
+  uri_endpoint: 'http://localhost:3000/{?subject}'
+  uri_root: 'http://localhost:3000/#dataset'
+  cache_backend:
+    provider: 'marmotta'
```

Comment:
It seems like we are only copying over one config file now. If this is the case, why not have it be the Repository one as it requires no additional running software? We can then provide documentation or samples on how to configure it against a Marmotta, Blazegraph, etc. instance.